src/requests/rocketchat.js

```diff
@@ -0,0 +1,208 @@
+const axios = require('axios')
+
+const chatPlatformAxios = axios.create({
+	baseURL: process.env.CHAT_PLATFORM_URL,
```

Comment:
@nevil-mathew if these are required envs add in environment and sample file as well